app/scripts/TiledPlot.js

```diff
@@ -457,26 +457,33 @@ class TiledPlot extends React.Component {
     ) return;
 
     // Get the total number of track that are expecting a tilesetInfo
-    const allTilesetInfos = Object.keys(this.trackRenderer.trackDefObjects)
+    const allTracksWithTilesetInfos = Object.keys(this.trackRenderer.trackDefObjects)
       // Map track to a list of tileset infos
       .map((trackUuid) => {
         const track = this.trackRenderer.trackDefObjects[trackUuid].trackObject;
         if (track.childTracks) {
           return track.childTracks.map(childTrack => childTrack.tilesetInfo);
```

Comment:
The line below this returns a track but this returns a tilesetInfo. Is that right?

Reply:
And not just a tilesetInfo but a list of tilesetInfos

Reply:
Oh good catch